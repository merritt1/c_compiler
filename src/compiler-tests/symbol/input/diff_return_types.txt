int main(void);
void main(void) {
    return;
}
