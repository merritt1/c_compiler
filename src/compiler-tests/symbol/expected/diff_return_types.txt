


***ERROR: Redefinition of function identifier 'main'


=============== PARSE TREE ===============
signed int (main(void));
void (main(void)) {
return;
}
================= SYMBOLS ================
Symbol Table ID: 0
  main -> function(signed int, [void])

